check_message.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -eu
+
+if [[ -v ZSH_NAME ]]; then
+  setopt BASH_REMATCH
+  setopt RE_MATCH_PCRE
+  setopt KSH_ARRAYS
+fi
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+source $DIR/validator.sh
+
+MESSAGE=$(<"$1")
+echo "checking commit message: '${MESSAGE}'"
+validate "$MESSAGE"
+
+echo "Message succesfully checked"
```

Comment:
Ditto, unneeded

Reply:
> I think this makes sense only as a `commit-msg` hook, not a pre-commit one.
> 
> I doubt the hook configuration itself works as-is, given that AFAIK `pre-commit` passes files to hooks, not commit SHAs

I was going to update the doc to explain you to set it up so it only runs over `commit-msg` ( basically just setting it up in `stages` in the config of `pre-commit`)

And I just figured that without COMMIT_VALIDATOR_ALLOW_TEMP, this prevents reverts and fixups... I'm adding it